test/spy-test.js

```diff
@@ -2226,6 +2226,34 @@
 
                 assert.equals(spy.length, 3);
             }
+        },
+
+        "getters and setters": {
+            "it is possible to spy getters": function () {
+                var object = {
+                    get myProperty() {
+                        return "fb41a645-24fb-4580-b4c0-17d71ecc1c74";
+                    }
+                };
+
+                var descriptor = sinon.spy(object, "myProperty", ["get"]);
+                var value = object.myProperty; // eslint-disable-line no-unused-vars
+
+                assert.equals(descriptor.get.callCount, 1);
+            }
+        },
+
+        "it is possible to spy setters": function () {
```

Comment:
The indentation is off here. This should have the same indentation as the test above it and the preceding closing brace should be moved below this test.